### PR TITLE
Update to Bevy 0.14.1

### PR DESCRIPTION
**Pull request by MathiasPius**
Thanks for putting this crate together, it's extremely helpful in understanding triplanar splatting!

I've taken the liberty of updating the code to work with Bevy 0.14.1.

P.S.: Is the repository unlicensed on purpose?

### CODE REVIEW

**Comment by bonsairobo** [comment]
@MathiasPius I'm glad you found it useful, and thank you for the contribution. Going to review the PR now.

> P.S.: Is the repository unlicensed on purpose?

No I simply neglected to add a license. I can put an MIT/Apache2 dual license on it soon.

File: rust-toolchain
Comment (bonsairobo) [inline rust-toolchain:1]: This breaks the `flake.nix`, but I will likely need to update it for bevy 0.14 anyway.

**Review by bonsairobo — APPROVED** [review]
Thank you so much for keeping this well-organized and easy to read.

I haven't tested the changes yet, but I will hopefully find time soon.

File: rust-toolchain
Comment (MathiasPius) [reply rust-toolchain:1]: My apologies, I don't use nix (yet) so I hadn't even considered it during the upgrade :)

**Pull request merged**